### DISCUSSION TITLE
Uploading document larger than 2Mo

### DISCUSSION THREAD

**Discussion by EricNahon** [pocketbase/pocketbase#2166] [Q&A] [answered]
First of all, congrats for this **awesome** project!
I'am migrating a Flutter web app from Firebase to Pocketbase (v0.13.4) and so far It has been a breeze.

The last issue I'm facing is uploading document larger than 2Mo. 
Although the default max file size is 5Mo, I've got an error with anything bigger than 2Mo, either using the admin GUI or via the Dart SDK.
"Something went wrong while processing your request" is the error I got via the admin GUI.
"..., isAbort: true, statusCode: 0, response: {}, originalError: XMLHttpRequest error.}" is the one via API
With any file smaller than 2Mo everything goes smoothly.

I've tried to increase the max file size to no avail. I've tried also different file types of various sizes.
I'am able to scp files of any size to the host (a vanilla Ubuntu server 22.04.2 LTS with NGINX acting as a proxy).
As I'am far from a linux guru I suspect I'am probably missing something with the host setup.

Has anyone experienced this issue?

Thank's for any help.

**Comment by ganigeorgiev** [comment, marked as answer]
Hm, it looks like a reverse proxy restriction. Does your server or location block has `client_max_body_size` set? If not, try to add `client_max_body_size 10M;`.

Alternatively, you can also start PocketBase with `--debug` to check for any additional error logs in the console, but judging from the `XMLHttpRequest` error I suspect a reverse proxy issue.

**Reply by EricNahon** [reply]
You're right, it was a reverse proxy restriction.
With the client_max_body_size 10M it works like a charm.

Thank you very much for your insanely fast and accurate answer!
